.travis.yml

```diff
@@ -2,7 +2,8 @@ language: node_js
 node_js:
   - "0.10"
   - "0.12"
-  - "iojs"
+  - "4"
+  - "stable"
```

Comment:
I like to keep this as stable (I wish we could reference LTE instead of 4 also).  This is the pattern used in all the gulpjs repos